src/tests/cli_perf.py

```diff
@@ -9,6 +9,8 @@
 from timeit import default_timer as perf_timer
 from cli_common import find_utility, run_proc, run_proc_fast, pswd_pipe, rnp_file_path, random_text, file_text, size_to_readable, raise_err
 import cli_common
+import inspect
+from optparse import OptionParser
```

Comment:
I think `optparse` was deprecated like 7 years ago, would prefer `argparse`, but not a big deal.

Reply:
agreed